llvm/test/MC/RISCV/corev/mem/invalid.s

```diff
@@ -0,0 +1,83 @@
+# RUN: not llvm-mc -triple=riscv32 %s 2>&1 \
+# RUN:        | FileCheck %s --check-prefix=CHECK-ERROR
+
+# Load instructions
+
+cv.lb t0, 0(t1!)
+# CHECK-ERROR: expected ')'
```

Comment:
Ah that's interesting. The instruction format error has precedence over the feature error. This is fine IMO, but its good to have  regression test for this.